azkaban-common/src/test/java/azkaban/executor/ContainerizedDispatchManagerTest.java

```diff
@@ -369,6 +367,7 @@ public void testSetFlowLock() throws Exception {
   @Test
   public void testDisablingQueueProcessThread() throws Exception {
     initializeContainerizedDispatchImpl();
+    this.containerizedDispatchManager.start();
```

Comment:
In disabled, shouldn't we stopping it?

Reply:
It is initialized in every test, and JUnit takes care of all the objects created in the test, including their threads.